src/ontology/mondo-edit.obo

```diff
@@ -108393,10 +108393,12 @@ xref: SCTID:192994008 {source="DOID:1827"}
 xref: SCTID:19598007 {source="MONDO:equivalentTo", source="DOID:1827"}
 xref: UMLS:C0270850 {source="MEDGEN:75725", source="MONDO:equivalentTo", source="MONDO:MEDGEN"}
 is_a: MONDO:0005027 {source="DOID:1827", source="EFO:0005917", source="MESH:D004829", source="NCIT:C3021"} ! epilepsy
+is_a: MONDO:0100574 ! generalized epilepsy
 intersection_of: MONDO:0005027 ! epilepsy
 intersection_of: has_characteristic MONDO:0700005 ! idiopathic
```

Comment:
The experts said that "idiopathic epilepsy" is actually not "idiopathic". It is a term to group 4 diseases. So we should update the SC axioms and logical definition. 
Also, this term should have less children (ie they should be only 4 children). 

IF needed (I don't know that we do), let's create another term that is "epilepsy of unknown etiology" (though I don't know that we absolutely need it). I need to review my notes.

Reply:
According to ILAE:

"The	2017 International	League	Against	Epilepsy	(ILAE)  classification suggested that the	term “genetic	generalized	epilepsies”	(GGEs)	be	used	for	the	broad	group	of	epilepsies	with	generalized	seizure	types	and	generalized	spike-	wave,	based	on	a	presumed	genetic	etiology	arising	from	twin	and	family	research	study	data.	It	suggested	that	the	term	IGE	could be	reserved	for	the	above	four	syndromes: [The	idiopathic	generalized	epilepsies	(IGEs)	have	historically	included	the	syndromes	childhood	absence	epilepsy	(CAE),	juvenile	absence	epilepsy	(JAE),	juvenile	myoclonic	epilepsy	(JME),	and	epilepsy	with	generalized	tonic–	clonic	seizures	alone	(GTCA)]."

- It turned out we actually had the class in Mondo already as generalized epilepsy, idiopathic (it is in NCIT, OMIM, others)
- I renamed it to ‘idiopathic generalized epilepsy’ (instead of generalized epilepsy, idiopathic) to match ILAE
- ILAE has four classes as children of ‘idiopathic generalized epilepsy’, which I reclassified
- there were already a few additional classes classified in this branch that come from OMIM - see [here](https://omim.org/phenotypicSeries/PS600669)
    - I am reviewing these children right now, I think some of them should be reclassified.

Reply:
- 'myoclonic-atonic epilepsy' is a child of 'myoclonic-astatic epilepsy' based on Orphanet:1942/btnt - I am not sure about this. I don't see 'myoclonic-astatic epilepsy in ILAE
- [DOID:0060475](https://www.ebi.ac.uk/ols4/ontologies/doid/classes?obo_id=DOID%3A0060475) defines this term as "An idiopathic generalized epilepsy characterized by onset of multiple seizure types in the first few years of life and associated with poor prognosis. Affected individuals have cognitive regression and intellectual disability and that has_material_basis_in heterozygous mutation in the SLC6A1 gene on chromosome 3p25." but according to ILAE, this is not one of the 'idiopathic generalized epilepsy' terms
- I'll bring this up at the next workshop